test/sanitize-html.test.js

```diff
@@ -105,19 +106,19 @@ describe('Sanitize HTML', () => {
 
   test('<iframe> tag to a spotify embed should not get removed', () => {
     const data = sanitizeHTML(
-      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https%3A%2F%2Fmedium.com%2F%40pedrofonsecadev%2Ftelescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
+      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https://medium.com/@pedrofonsecadev/telescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
     );
     expect(data).toBe(
-      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https%3A%2F%2Fmedium.com%2F%40pedrofonsecadev%2Ftelescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
+      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https://medium.com/@pedrofonsecadev/telescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
```

Comment:
These aren't technically valid URLs (with the :// included like this).  I guess if it passes, it's OK, but it's not ideal.

Reply:
Would it be better to leave them with their escaped characters as it was originally? For readability purposes, this is currently fine but the escaped characters is right. I can change it back quickly

Reply:
I think in the escaped form is better, yes.